### PR TITLE
Add new files to the 'data/examples' directory

### LINKED ISSUE #76: Add new files to `data/examples`
From https://zenodo.org/records/10647894, there are : 
- DNA_map_jump.pdb
- DNA_map_reor.pdb
- DNA_start.gro
- DNA_start_nowat.gro
- RNA_map_jump.pdb
- RNA_map_reor.pdb
- RNA_start.gro
- RNA_start_nowat.gro

There is multiple PDB and GRO file, should we add all of it ?

### PR DESCRIPTION
**Pull request by KarinDuong**
From https://zenodo.org/records/10647894
close #76

### CODE REVIEW

**Comment by pierrepo** [comment]
Could we keep only `DNA_start.gro` and `RNA_start.gro`? All other structures are kind of redundant.

File: grodecoder.py [hunk 1]
```diff
@@ -1317,7 +1317,7 @@ def is_met(graph: nx.classes.graph.Graph) -> bool:
 def is_nucleic_acids(graph: nx.classes.graph.Graph) -> bool:
     set_key_amino_acid_mda = set(mol_def.NUCLEIC_ACIDS.keys())
```
Comment (pierrepo) [inline grodecoder.py:1318]: The variable name `set_key_amino_acids_mda` does not seem appropriate. Could you use `nucleic_acid_resnames` instead?

**Comment by pierrepo** [comment]
We don't need `DNA_start_nowat.gro` and `RNA_start_nowat.gro`

**Comment by KarinDuong** [comment]
I already remove them

**Comment by pierrepo** [comment]
OK. Ready for merge?

**Comment by KarinDuong** [comment]
Yes ready to merge

**Pull request merged**